plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleConfig.java

```diff
@@ -142,6 +144,20 @@ public OracleConfig setInactiveConnectionTimeout(Duration inactiveConnectionTime
         return this;
     }
 
+    @NotNull
+    public Duration getConnectionWaitDuration()
+    {
+        return connectionWaitDuration;
+    }
+
+    @Config("oracle.connection-pool.wait-duration")
+    @ConfigDescription("How long to wait for a used connection to be released by a client")
```

Comment:
I found "specifies the maximum amount of time an application request will wait to obtain an available connection from the pool if all connections are currently in use", could you refer this can rephrase it